File: C# .Net/JDI UI Framework/JDI/Web/Selenium/Elements/Composite/WebSite.cs
using System;
using Epam.JDI.Core.Base;
using Epam.JDI.Web.Selenium.Base;
using Epam.JDI.Web.Settings;

namespace Epam.JDI.Web.Selenium.Elements.Composite
{
    public class WebSite : Application
    {
        public static void Init(Type site)
        {
            new WebCascadeInit().InitStaticPages(site, WebSettings.WebDriverFactory.CurrentDriverName);
            CurrentSite = site;
        }
        public static T Init<T>(Type site, string driverName) where T : Application
        {
            return new WebCascadeInit().InitPages<T>(site, driverName);
        }
    }
}

File: Python/JDI/web/selenium/elements/composite/web_site.py
from JDI.core.interfaces.application import Application
from JDI.web.selenium.elements.web_cascade_init import WebCascadeInit
from JDI.web.selenium.settings.web_settings import WebSettings


class WebSite(Application):

    @staticmethod
    def _init(site):
        WebCascadeInit().init_site_page(site)

    @staticmethod
    def init(site, driver_name):
        WebSettings.use_driver(driver_name)
        WebCascadeInit().init_site_page(site)
